code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -235,20 +235,14 @@
 	return amount
 
 /datum/reagents/proc/trans_to_holder(var/datum/reagents/target, var/amount = 1, var/multiplier = 1, var/copy = 0) // Transfers [amount] reagents from [src] to [target], multiplying them by [multiplier]. Returns actual amount removed from [src] (not amount transferred to [target]).
-
```

Comment:
Is the deleting of newlines here intentional? Some of the ones you've deleted make the proc harder to read.